openapi/openapi/src/main/scala/endpoints4s/openapi/model/OpenApi.scala

```diff
@@ -1,21 +1,22 @@
 package endpoints4s.openapi.model
 
-import java.io.Serializable
-
-import endpoints4s.{Encoder, Hashing}
 import endpoints4s.algebra.{ExternalDocumentationObject, Tag}
+import endpoints4s.{Encoder, Hashing}
 
-import scala.collection.mutable
+import java.io.Serializable
+import scala.collection.{Map, mutable, immutable}
```

Comment:
This import changed all the places where we were previously using `Map` (which was `scala.collection.immutable.Map`). I suggest removing the import and fully qualifying the name `scala.collection.Map` instead, where we use it.